<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    border: 0px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>744</width>
      <height>503</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="userWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>311</width>
         <height>501</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>311</width>
         <height>501</height>
        </size>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>60</width>
          <height>500</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(15, 143, 15);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>0</y>
          <width>250</width>
          <height>65</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(179, 179, 179);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTableWidget" name="userListTableWidget">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>65</y>
          <width>250</width>
          <height>435</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>250</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="avatarLabel">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>17</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="userNameLabel">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>用户名</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>17</width>
               <height>5</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="chatWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>431</width>
         <height>501</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>431</width>
         <height>501</height>
        </size>
       </property>
       <widget class="QLabel" name="friendNameLabel">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>1</y>
          <width>121</width>
          <height>62</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>432</width>
          <height>65</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(190, 190, 190);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTextBrowser" name="chatTextBrowser">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>65</y>
          <width>431</width>
          <height>300</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="transPushButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>375</y>
          <width>31</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>传</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>365</y>
          <width>432</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(202, 202, 202);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QTextEdit" name="chatTextEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>403</y>
          <width>431</width>
          <height>97</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="sendPushButton">
        <property name="geometry">
         <rect>
          <x>355</x>
          <y>476</y>
          <width>75</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>发  送(S)</string>
        </property>
       </widget>
       <zorder>label_4</zorder>
       <zorder>chatTextBrowser</zorder>
       <zorder>label_5</zorder>
       <zorder>chatTextEdit</zorder>
       <zorder>sendPushButton</zorder>
       <zorder>friendNameLabel</zorder>
       <zorder>transPushButton</zorder>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
